### ISSUE TITLE
Problems when increasing column size by dragging

### ISSUE THREAD

**Issue by telamonian** [jpmorganchase/regular-table#37]
### Steps to Reproduce:
1. Open one of the examples
2. Click to drag one of the column header resizers
3. Drag resizer to the right (try to make column larger)

### Expected Result:
Smoothly animated column resize

### Actual Result:
Many console errors of the form:

```
TypeError: Cannot read property 'style' of undefined
    at HTMLElement._on_resize_column_move (events.js:178)
    at async HTMLElement.i.value (utils.js:101)
```

In some (most?) cases the column resize will still succeed, but the errors seem to prevent the animation/visual feedback from working correctly.

### Environment:
osx 10.14, chromium 76

### Additional Context:
The issue seems to be the recent removal of `vcidx` from th metadata. One possible resolution would be to remove the second "optimized" branch from this conditional:

https://github.com/jpmorganchase/regular-table/blob/f60a572904ee753624f9516182ff23a6562d0562/src/js/events.js#L168-L181

**Comment by texodus** [comment]
Thanks for the detailed report!  This is fixed and will be released in 0.0.6.

**Comment by telamonian** [comment]
Awesome. Thanks!
